### ISSUE TITLE
Requester Reviews Syntax Errors in Firefox

### ISSUE THREAD

**Issue by neocryptek** [brandonhellman/mturk-suite#69]
I'm seeing the requester reviews hover popup failing to show up on hover on some requesters. Looking at the javascript console I see these errors on loading the main HIT search results page. There seems to be an errors whenever I find I can't hover, so I think they're related.

## Expected Behavior
Requester review popup should show up on hover for all requesters on the page.

## Current Behavior
```
SyntaxError: missing } after property list[Learn More]
worker.mturk.com:1:467
<anonymous>
moz-extension://ca9fc9de-2c7b-5a4a-ba29-82e812cc32df/mturk/requester-reviews.js:111:7
next self-hosted:1211:9
note: { opened at line 1, column 43
worker.mturk.com:1:43

SyntaxError: missing } after property list[Learn More]
worker.mturk.com:1:461
<anonymous>
moz-extension://ca9fc9de-2c7b-5a4a-ba29-82e812cc32df/mturk/requester-reviews.js:111:7
next self-hosted:1211:9
note: { opened at line 1, column 43
worker.mturk.com:1:43

SyntaxError: missing } after property list[Learn More]
worker.mturk.com:1:97
<anonymous>
moz-extension://ca9fc9de-2c7b-5a4a-ba29-82e812cc32df/mturk/requester-reviews.js:111:7
next self-hosted:1211:9
note: { opened at line 1, column 43
worker.mturk.com:1:43

SyntaxError: missing } after property list[Learn More]
worker.mturk.com:1:431
<anonymous>
moz-extension://ca9fc9de-2c7b-5a4a-ba29-82e812cc32df/mturk/requester-reviews.js:111:7
next self-hosted:1211:9
note: { opened at line 1, column 43
worker.mturk.com:1:43

SyntaxError: missing } after property list[Learn More]
worker.mturk.com:1:438
<anonymous>
moz-extension://ca9fc9de-2c7b-5a4a-ba29-82e812cc32df/mturk/requester-reviews.js:111:7
next self-hosted:1211:9
note: { opened at line 1, column 43
worker.mturk.com:1:43
```

Is there some more useful information I can provide? Here's the code it is pointing to:
https://github.com/Kadauchi/mturk-suite/blob/master/mturk/requester-reviews.js#L111

Some requesters I've seen that it continuously happens on is:
https://worker.mturk.com/requesters/A3UP7OA0OAUQD1/projects?ref=w_pl_prvw
Neeraj Agarwal
https://worker.mturk.com/requesters/A35J4BZNLH89XW/projects?ref=w_pl_prvw
Gregor Weiß
https://worker.mturk.com/requesters/A2WTYRUG14FBF/projects?ref=w_pl_prvw
Conor O'Dea
https://worker.mturk.com/requesters/A1OW488VND8MN8/projects?ref=w_pl_prvw
RDG Learn
https://worker.mturk.com/requesters/A2K9SD253VNPRD/projects?ref=w_pl_prvw
Trident

I don't see an obvious pattern to them--some have special characters, some have non-ascii characters, some have neither...

For example if you search all HITs for "conor" to get all 4 that requester has, there will be 4 HITs listed, and none of them will show the popup. There's only 2 errors in the javascript console though...

## Possible Solution
Need to insert a closing } somewhere :)

## Steps to Reproduce (for bugs)
1. Load main HIT search page
2. Hover over every requester in the page
3. Notice that some requesters don't show the reviews popup on hover
4. Check javascript console for errors

## Context
Viewing requester reviews on hover in main HIT search page.

## Your Environment
MTurk Suite 2.2.3.1 extension installed through store
Browser: Firefox 60.0.0 (64-bit) (MacOS 10.11.6)

I tried disabling Tampermonkey so no other scripts could possibly be conflicting with MTS, still was able to reproduce it.

**Comment by brandonhellman** [comment]
You are correct and they are related. It's because I wasn't escaping html characters from requester names or  titles so when the popup code gets injected, it breaks.

Fix is done and will ship with 2.3.0.

**Comment by neocryptek** [comment]
Thanks!
